lein-finagle-clojure/src/leiningen/finagle_clojure.clj

```diff
@@ -1,16 +1,38 @@
 (ns leiningen.finagle-clojure
   (:require [leiningen.javac]
             [robert.hooke]
-            [clojure.java.io :as io]))
+            [clojure.java.io :as io]
+            [leiningen.core.classpath :as classpath]
+            [cemerick.pomegranate :as pom])
+  (:import (java.io File)
+           (java.net URL)))
 
-(defn- thrift-files
+(defn- find-thrift-files
   [project-root source-path]
   (->> source-path
     (io/file project-root)
     (file-seq)
     (filter #(and (.isFile %) (.endsWith (.getName %) ".thrift")))
     (map #(.getPath %))))
 
+(defn- scrape-includes
+  "Returns the filenames of thrift files this code tries to include."
+  [thrift-code]
+  (map second (re-seq #"include \"(.+\.thrift)\"" thrift-code)))
+
+(def ^:private target-temp-dir
+  "target/thrift-include")
```

Comment:
Instead of hardcoding `target` here, you should look up the `:target-path` in the project map.